python-lib/dku_utils.py

```diff
@@ -19,7 +22,30 @@ def get_endpoint_parameters(configuration):
         "requests_per_minute",
         "pagination_type",
         "next_page_url_key",
-        "top_key", "skip_key"
+        "top_key", "skip_key", "maximum_number_rows"
     ]
     parameters = {endpoint_parameter: configuration.get(endpoint_parameter) for endpoint_parameter in endpoint_parameters if configuration.get(endpoint_parameter) is not None}
     return parameters
+
+
+def parse_keys_for_json(items):
+    ret = {}
+    for key in items:
+        value = items.get(key)
+        if isinstance(value, dict) or isinstance(value, list):
+            ret.update({key: json.dumps(value)})
+        elif value is None:
+            continue
+        else:
+            ret.update({key: value})
+    return ret
+
+
+def get_value_from_path(dictionary, path, default_reply=None):
+    ret = dictionary
```

Comment:
to not copy only the reference, you can use `ret = copy.deepcopy(dictionary)` from the copy package